packages/client/src/components/add-liquidity/add-liquidity-v3.tsx

```diff
@@ -404,6 +404,13 @@ export const AddLiquidityV3 = ({
             }
         }
 
+        if (Math.abs(upperBoundTick - lowerBoundTick) == uniPool.tickSpacing) {
+            upperBoundTick +=
+                upperBoundTick > lowerBoundTick
```

Comment:
The upper bound tick should always be greater than than the lower bound tick. We should be bailing out earlier in the function if that isn't true.

Reply:
In this code the upperBoundTick is the tick calculated by upperBound, and the lowerBoundTick is the tick calculated by lowerBound.
And we check what tick is lower when call Position.fromAmounts.
![image](https://user-images.githubusercontent.com/60648300/129618607-2b062d74-9a77-4844-a91f-67bdc9bdfbc9.png)

Reply:
Yeah I think having a code path where the upper tick moves under the lower tick can lead to confusing outcomes

Reply:
Yeah, it is a lilttle confusing.
Below that code, there are the code to calculate priceLower and priceUpper.
So we will have lowerBound, lowerBoundTick, priceLower as one pair, same for upper.
When we have infinity errors at last time, ticks were sorted and priceLower was calculated by lower tick sorted in ticks.
It was wrong and it causes errors.
The priceLower should be calculated by the tick of lowerBound, not by the lower tick in sorted ticks.